zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/test/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProviderTest.java

```diff
@@ -56,6 +56,7 @@ public void setup() throws Exception {
         CollectorRegistry.defaultRegistry.clear();
         provider = new PrometheusMetricsProvider();
         Properties configuration = new Properties();
+        configuration.setProperty("httpHost", "127.0.0.1"); // ephemeral port
         configuration.setProperty("httpPort", "0"); // ephemeral port
```

Comment:
ephemeral hostname? :)

Reply:
It's a mistake, it should be "local host for test"